src/components/users/user-profile.scss

```diff
@@ -1,51 +1,75 @@
-@import "../style/common/constants";
+@import "../../style/common/constants";
 
 
 
 /*------------------------------------*\
-  #LEFT-SIDE
+  #USER-PROFILE
 \*------------------------------------*/
-.left-menu {
-    background-color: $white-left-side;
+.user-profile {
+    background-color: $white;
     width: 30vw;
     text-align: center;
     position: relative;
 }
 
-.left-menu__profile-picture {
+.user-profile__profile-picture {
     align-content: center;
     align-items: center;
     display: inline-block;
 }
 
-.left-menu__options {
+.user-profile__options {
     padding-top: 2em;
     padding-left: 1.5em;
     position: absolute;
 }
 
-.left-menu__app-title {
+.user-profile__app-title {
     font-family: $font-family-bold;
     font-size: 1.5em;
     line-height: 1.5em;
     letter-spacing: 0.2em;
 }
 
-.left-menu__username {
+.user-profile__username {
     font-family: $font-family-bold;
     font-size: 1em;
     line-height: 1em;
     letter-spacing: 0.1em;
 }
 
-.left-menu__edit-logout {
+.user-profile__edit-logout {
     font-family: $font-family-regular;
     font-size: 0.8em;
     line-height: 1em;
     letter-spacing: 0.1em;
 }
 
-.left-menu__matches {
+.logout__btn-text {
+    font-family: $font-family-regular;
+    font-size: 1em;
+    line-height: 1em;
+    letter-spacing: 0.1em;
+    background: none!important;
+    border: none;
+    padding: 0!important;
+    font-family: arial, sans-serif;
+    cursor: pointer;
+}
+
+.logout__btn-text:hover {
+    // Underline and turn the button blue on hover
+    text-decoration: underline;
+    color: $dodger-blue;
+}
+
+.logout__btn-text:focus{
+    // Eliminate border when the button is clicked
+    border-color:transparent!important;
```

Comment:
try to avoid use !important as long as possible, it's considered a bad practice since could lead in side effects. There will be some special cases when you won't have another way to go, but in general you will be able to avoid them just restructuring your css.

Reply:
And what is a good practice to remove the border of a button?

Reply:
I guess that playing around with border-color, outline and box-shadow css properties you will be able to achieve the expected behavior, I'm discouraging the use of **!important** . doesn't it work if you remove it? Maybe you need it to override the browser default style under _focus_ state, in this case it's ok leave it, but you should be able to set background and padding in lines 52 and 55 without a !important tag.